TTHAnalysis/python/plotter/ttH-multilepton/mca-includes/mca-2lss-sigprompt.txt

```diff
@@ -1,8 +1,16 @@
-ttH+    : TTHnobb_fxfx : 0.5071*0.418 : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge; FillColor=ROOT.kOrange+10
+ttH+    : TTHnobb_pow : 0.5071*0.418 : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge; FillColor=ROOT.kOrange+10
```

Comment:
Please don't commit changes that would affect everyone, just leave them in the working directory.
This needs to be reverted.

Also, this is just wrong (see comment on diff line 34), as you left the conflicts on!

Reply:
maintained

Reply:
V6 have now fxfx, so the final version shoudl have fxfx